src/plugins/dashboard/public/dashboard_actions/copy_to_dashboard_action.tsx

```diff
@@ -48,7 +49,8 @@ const apiIsCompatible = (api: unknown): api is CopyToDashboardAPI => {
     apiHasUniqueId(api) &&
     apiHasParentApi(api) &&
     apiIsOfType(api.parentApi, DASHBOARD_CONTAINER_TYPE) &&
-    apiPublishesSavedObjectId(api.parentApi)
+    apiPublishesSavedObjectId(api.parentApi) &&
+    !apiHasExpandedPanel(api.parentApi)
```

Comment:
nit: Rather than making a new type guard, it might be easier to call the existing one:

``` typescript
!(apiCanExpandPanels(api.parentApi) && api.parentApi.expandedPanelId.getValue())
```